src/main/scala/firrtl/util/BackendCompilationUtilities.scala

```diff
@@ -93,8 +89,22 @@ trait BackendCompilationUtilities {
                     cppHarness: File
                   ): ProcessBuilder = {
     val topModule = dutFile
-    val command = Seq("verilator",
-      "--cc", s"$dutFile.v") ++
+
+    val blackBoxVerilogList = {
+      val list_file = new File(dir, firrtl.transforms.BlackBoxSourceHelper.FileListName)
+      if(list_file.exists()) {
+        Seq("-f", list_file.getAbsolutePath)
+      }
+      else {
+        Seq.empty[String]
+      }
+    }
```

Comment:
I'm not convinced that this is the right way to do this. Looking at the function, this adds some magical arguments from a file with a certain name, would it not be better to have the caller pass the filename to the function itself?